mkdocs.yml

```diff
@@ -1,6 +1,6 @@
 # Project information
-site_name: ""
-site_url: https://w3f.github.io/polkadot-wiki-mkdocs
+site_name: "Polkadot Wiki"
+site_url: http://localhost:8888
```

Comment:
Why is this localhost?

Reply:
Ah yeah this shouldn't be, I think I was testing something to do with redirects, will change before merging